content/millennium/dstu2.md

```diff
@@ -332,6 +332,26 @@ $ curl -i -H "Accept: application/json+fhir" "https://fhir-open.sandboxcerner.co
 In this example, MedicationOrder is the FHIR<sup>®</sup> standard resource being accessed, while `patient` and `status` 
  are passed in the query string.
 
+ ### Resource Identity
+
+ Please note that no ids or identifiers in Millennium EHR are intended to be used outside of the context of their complete URL. A complete URL comprises of [service root url](http://hl7.org/fhir/dstu2/http.html#general), the resource, and parameters (if any).
```

Comment:
grammar nit: 'comprises of' sounds odd to me, can we switch to either 'comprises the' or 'is comprised of the' and add 'the' before parameters?